_ZN5boost6system16generic_categoryEv missing symbol

Hi,

when building pristine 0.5.6 on Ubuntu Cosmic (18.10) in armhf, I get this linker error:
```
  CXXLD    snapper
/usr/bin/ld: ../snapper/.libs/libsnapper.so: undefined reference to symbol '_ZN5boost6system16generic_categoryEv'
/usr/bin/ld: /usr/lib/gcc/arm-linux-gnueabihf/7/../../../arm-linux-gnueabihf/libboost_system.so: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
Makefile:639: recipe for target 'snapper' failed
make[3]: *** [snapper] Error 1
```
For some reason it doesn't happen in other architectures, i.e., there the dependency on boost is apparently found correctly.
This patch fixed the build for me:
```diff
--- a/dbus/Makefile.am
+++ b/dbus/Makefile.am
@@ -12,4 +12,4 @@ libdbus_la_SOURCES =					\
 	DBusMainLoop.cc		DBusMainLoop.h
 
 libdbus_la_LIBADD = $(DBUS_LIBS)
-
+libdbus_la_LDFLAGS = -lboost_system -lboost_thread

```
Does this make sense for you upstream?
I noticed something similar was done once in the past in https://github.com/openSUSE/snapper/commit/33fbde9412d5eafb9f94aaff6b1be10bcda14951

--- a/dbus/Makefile.am
+++ b/dbus/Makefile.am
@@ -12,4 +12,4 @@ libdbus_la_SOURCES =					\
 	DBusMainLoop.cc		DBusMainLoop.h
 
 libdbus_la_LIBADD = $(DBUS_LIBS)
-
+libdbus_la_LDFLAGS = -lboost_system -lboost_thread

